### ISSUE TITLE
@graphql-codegen/visitor-plugin-common@1.0.6" has incorrect peer dependency "graphql@14.2.0"

### ISSUE THREAD

**Issue by jonaskello** [dotansimha/graphql-code-generator#1637]
**Describe the bug**

@graphql-codegen/visitor-plugin-common@1.0.6" has incorrect peer dependency "graphql@14.2.0"

**To Reproduce**

Check package.json [here](https://github.com/dotansimha/graphql-code-generator/blob/c75ea46773e5f7c46c458e92bc9378f070c8cc37/packages/plugins/visitor-plugin-common/package.json#L17). It should probably allow a wider range of the graphql package?

```
"peerDependencies": {
    "graphql": "14.2.1"
  },
```

I think it should be something like this:

```
"peerDependencies": {
  "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0"
}
```

Or at least it should allow `^14.2.1`.

See [this article](https://medium.com/@leeb/graphql-js-preparing-for-v14-0-0-839f823c144e) for more guidance on this.

**Comment by dotansimha** [comment]
Hi @jonaskello ! Thank you for reporting it, I'll fix it in the next release.

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1663

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)
